headlamp-k8s/headlamp
# Notarize Mac app using ESRP

We are moving the app in Apple from the legacy Kinvolk developer account to Microsoft's.
For that, we have to use the ESRP tool that is also used to sign the Windows binaries.

- [X] Replace the Mac signing mechanism by ESRP
- [X] Use just one ESRP script for Windows and Mac signing

How to test:
- In the actions, run the Windows and Mac artifacts with the Sign/Notarize option checked: install the resulting binaries and verify that the OS doesn't complain about them being unsigned
